cilinghir.alexandr/homework/lesson4_homework.rb

```diff
@@ -0,0 +1,48 @@
+
+@sep = "--------------------------------------------------------------------\n"
+puts "Float/integer"
+puts @sep
+@var_1 = -123.to_i
+puts "-123 it is odd? - its #{@var_1.odd?}"
+puts "\n"
+# Question_2
+@var_2 = '6893'.next.to_i
+puts "Next number of 6893 -> #{@var_2}"
+puts "\n"
+# Question_3
+@var_3 = 0.to_i-1
```

Comment:
Again, no need to convert to `Integer`. Also, check out `prev` method, which is similar to `next`